cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -37,9 +38,10 @@ set(config_module_list
 	drivers/ms5525_airspeed
 	drivers/sdp3x_airspeed
 	drivers/frsky_telemetry
+
 	modules/sensors
 	#drivers/mkblctrl
-	drivers/px4flow
+	#drivers/px4flow
```

Comment:
why commenting the px4flow module?

Reply:
I should have stated, these changes are only made to get everything to fit and the right things to be included for testing. I will remove these changes once this is ready to be merged.